apps/emqx/src/emqx_persistent_session_ds_router.erl

```diff
@@ -52,7 +53,7 @@ init_tables() ->
     ok = mria:create_table(?PS_ROUTER_TAB, [
         {type, bag},
         {rlog_shard, ?PS_ROUTER_SHARD},
-        {storage, ram_copies},
+        {storage, disc_copies},
```

Comment:
can we change schema like this?

Reply:
This is behind a feature flag:

- https://github.com/emqx/emqx/blob/cf334d55422f16f68140e78bda63d12f236d605f/apps/emqx/src/emqx_schema.erl#L321-L328
- https://github.com/emqx/emqx/blob/cf334d55422f16f68140e78bda63d12f236d605f/apps/emqx/src/emqx_persistent_session_ds.erl#L59-L66

And no one should be relying on this feature.